primefaces/src/main/java/org/primefaces/component/export/ExcelOptions.java

```diff
@@ -45,6 +47,8 @@ public class ExcelOptions implements ExporterOptions {
 
     private boolean stronglyTypedCells = true;
 
+    private DecimalFormat decimalFormat;
+
```

Comment:
Can you add a note in `\docs\12_0_0\gettingstarted\whatsnew.md` stating this new option exists in ExcelOptions for DataExporter.

Can you update `\docs\12_0_0\components\dataexporter.md` at the bottom where it shows the custom excel options add DecimalFormat in that example?